src/main/resources/assets/splatcraft/lang/ru_ru.json

```
@@ -0,0 +1,429 @@
+  "weaponTab.splatcraft:weapon_workbench_tabs/splatlings": "Брызгомёты",
+  "weaponTab.splatcraft:weapon_workbench_tabs/brellas": "Зонты",
+  "weaponTab.splatcraft:weapon_workbench_tabs/sub_weapons": "Запасные оружия",
+  "weaponTab.splatcraft:weapon_workbench_tabs/special_weapons": "Особые оружия",
+  "weaponTab.splatcraft:weapon_workbench_tabs/ink_tanks": "Баллоны",
+
+  "weaponRecipe.splatcraft:weapon_workbench/aerospray": "Аэроспрей",
+  "weaponRecipe.splatcraft:weapon_workbench/n-zap": "N-ZAP",
+  "weaponRecipe.splatcraft:weapon_workbench/e_liter": "Э-литр",
+  "weaponRecipe.splatcraft:weapon_workbench/bamboozler_14": "Бамбук 14-1",
+
+  "weaponType.original": "Оригинальный",
+  "weaponType.classic": "Классический",
+  "weaponType.custom": "<<Понт>>",
+  "weaponType.deco": "<<Деко>>",
+  "weaponType.kensa": "<<от С.Кусаки>>",
+  "weaponType.hero": "Геройский",
+  "weaponType.replica": "Реплика",
+  "weaponType.hero_replica": "Геройский (Реплика)",
+  "weaponType.ultimate": "Ультимативный",
+  "weaponType.tentatek": "Шуччи",
+  "weaponType.krak_on": "<<Кра-Кен>>",
+  "weaponType.enperry": "<<Кальмар-Кляр>>",
+  "weaponType.neo": "Нео",
+  "weaponType.wasabi": "<<Васаби>>",
+  "weaponType.coro_coro": "<<КороКоро>>",
+  "weaponType.bento": "<<Бэнто>>",
+  "weaponType.grim": "<<Мрак>>",
+  "weaponType.permanent": "Водостойкая",
+  "weaponType.soda": "Содовая",
+  "weaponType.kelp": "<<Ламинарий>>",
+  "weaponType.3k": "55",
+  "weaponType.4k": "95",
+  "weaponType.mg": "",
+  "weaponType.rg": "ДЕЛЮКС",
+  "weaponType.pg": "ИДЕАЛ",
+  "weaponType.mk1": "-I",
+  "weaponType.mk2": "-II",
+  "weaponType.mk3": "-III",
+  "weaponType.83": "'83",
+  "weaponType.85": "'85",
+  "weaponType.89": "'89",
+
+  "subtitle.shooter_firing": "Краскомат стреляет",
+  "subtitle.blaster_firing": "Выстрел бластера",
+  "subtitle.blaster_explosion": "Взрыв бластера",
+  "subtitle.roller_fling": "Стрельба валиком",
+  "subtitle.roller_roll": "Валик катится",
+  "subtitle.brush_fling": "Кисть стреляет",
+  "subtitle.brush_roll": "Кисть бежит",
+  "subtitle.charger_charge": "Нагнетатель заряжается",
+  "subtitle.charger_ready": "Нагнетатель заряжен",
+  "subtitle.charger_shot": "Нагнятатель стреляет",
+  "subtitle.dualie_firing": "Спуртомёты стреляют",
+  "subtitle.dualie_dodge": "Спуртомёты кувыркаются",
+  "subtitle.slosher_shot": "Ведроган стреляет",
+  "subtitle.sub_throw": "Запасное оружие брошено",
+  "subtitle.sub_detonating": "Бомба детонируется",
+  "subtitle.sub_detonate": "Взорвалась бомба",
+  "subtitle.remote_use": "Использован пульт",
+  "subtitle.power_egg_can_open": "Консервы открыты",
+  "subtitle.squid_bumper.pop": "Цель лопается",
+  "subtitle.squid_bumper.respawning": "Цель появляется",
+  "subtitle.squid_bumper.ready": "Цель готова",
+  "subtitle.no_ink": "Нет краски!",
+  "subtitle.no_ink_sub": "Нет краски!",
+  "subtitle.squid_transform": "Трансформация",
+  "subtitle.squid_revert": "Отмена",
```

couldn't find a better translation
```suggestion
  "subtitle.squid_revert": "Обратная трансформация",
```